docs/components/modeler/desktop-modeler/element-templates/about-templates.md

```diff
@@ -10,7 +10,7 @@ Element templates are currently available in BPMN diagrams only.
 
 # Overview
 
-Element templates are a way to extend [Desktop Modeler](https://camunda.org/bpmn/tool/) with domain-specific diagram elements, such as service and user tasks.
+Element templates are a way to extend our Modelers with domain-specific diagram elements, such as service and user tasks. They support Camunda 7 and Camunda 8 and are the foundation of [Camunda 8 Connector Templates](../../../../../components/integration-framework/connectors/custom-built-connectors/connector-templates/).
```

Comment:
Same question -- should we apply these to the older versions of the docs too? I don't have a strong opinion either way, but want to raise it in case you do.

Reply:
Yes. Is equally applicable to 8.1 and 8.0 branches; not supported in < 8.0 though.